app/controllers/business_admin/businesses_controller.rb

```diff
@@ -1,10 +1,6 @@
 class BusinessAdmin::BusinessesController < BusinessAdminController
   def show
-    @business = current_user.business
-    @statuses = Job.pluck(:status).uniq
-    @jobs = @business.jobs
-    @business.image_url ||= Faker::Company.logo
-    @business.save
+    @presenter = BusinessAdminDashboardPresenter.new(current_user)
```

Comment:
Maybe the name of this should reflect what type of presenter object it is so other devs know what they are looking at even if they only see our view.